src/main/java/adris/altoclef/commandsystem/CommandExecutor.java

```diff
@@ -18,7 +18,7 @@ public CommandExecutor(AltoClef mod, String commandPrefix) {
         _commandPrefix = commandPrefix;
     }
 
-    public void RegisterNewCommand(Command command) throws InvalidKeyException {
+    public void registerNewCommand(Command command) throws InvalidKeyException {
```

Comment:
Yeah I should have renamed these long ago, like the moment I imported it in, so this is objectively better. But just like Debug.js, this is making a broad sweeping change that has little to do with your killaura changes. I'd suggest either deleting this and making an issue, or making a different PR.